src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -576,14 +567,20 @@ FancyZones::OnKeyDown(PKBDLLHOOKSTRUCT info) noexcept
             }
         }
     }
-    // Temporarily disable Win+Ctrl+Number functionality
-    //else if (m_inMoveSize && (info->vkCode >= '0') && (info->vkCode <= '9'))
-    //{
-    //    // This allows you to cycle through ZoneSets while dragging a window
-    //    Trace::FancyZones::OnKeyDown(info->vkCode, win, false /*control*/, true /*inMoveSize*/);
-    //    CycleActiveZoneSet(info->vkCode);
-    //    return false;
-    //}
+
+    if (!shift && win && ctrl && alt && m_settings->GetSettings()->quickLayoutSwitch)

```

Comment:
Why `!shift` is checked here? If this prevents from switching while dragging, then we have to check a non-primary mouse button too with a corresponding setting.

Reply:
It's OK since we are going to add support for switching layout just pressing a number while `m_windowMoveHandler.IsDragEnabled()` is `TRUE`.